lib/email.rb

```diff
@@ -0,0 +1,13 @@
+module Email
+end
+
+require 'ERB'
+require 'yaml'
+
+require_relative './email/key'
+require_relative './email/locale'
+require_relative './email/middleman'
+require_relative './email/store'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.